en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md

```diff
@@ -0,0 +1,146 @@
+# Deploy an API to Microgateway
+
+There are two ways to add an API to the Microgateway.
+
+1. [Via API Manager](#via-api-manager)
+
+    By enabling the Control Plane Event Hub in the Microgateway configuration file, the APIs deployed in API Manager will automatically be available in Microgateway. 
+
+2. [Via the CLI tool (apictl)](#via-the-cli-tool-apictl)
+
+    If you are not using WSO2 API Manager yet, or only want to tryout microgateway, simply use the CLI tool [apictl](https://github.com/wso2/product-apim-tooling/releases) to host your API.
```

Comment:
```suggestion
    If you are not using WSO2 API Manager yet, or only want to try out Microgateway, simply use the CLI tool [apictl](https://github.com/wso2/product-apim-tooling/releases) to host your API.
```
Shouldn't it be Choreo Connect and not Microgateway?

Reply:
We thought of renaming all the places in one go